package.json

```diff
@@ -33,13 +33,16 @@
     "eslint": "eslint --ext .js *.js lib tests"
   },
   "dependencies": {
+    "add": "^2.0.6",
```

Comment:
Why?

Reply:
My mistake, I will remove it.
Miss the basic self review 🥲